firebase-perf/ktx/gradle.properties

```diff
@@ -1,2 +1,3 @@
 android.enableUnitTestBinaryResources=true
-
+version=20.0.7-eap
```

Comment:
You can revert the changes to this file. It's only changed by the tool so there wouldn't be any conflict.

Reply:
Done!